.circleci/config.yml

```diff
@@ -6,19 +6,18 @@ jobs:
       - image: rsrchboy/mdl
     steps:
       - checkout
-      - run: mdl .
+      - run: mdl --style ./.circleci/.mdlrc .
 
-  flake8:
+  black:
     docker:
-      - image: python:2.7
+      - image: 667005031541.dkr.ecr.us-west-1.amazonaws.com/black:18.9b0
```

Comment:
Can you use a public image? We don't want to use private stuff in our public repo to prevent leaking (anyone can fork, open a PR, and dump some env vars)